src/service/fetchData.js

```diff
@@ -0,0 +1,5 @@
+export function getData(setData) {
```

Comment:
It is not a good practice to pass a prop setter in a function.
instead, pass a custom function or return the data.